.github/workflows/connectors_format.yml

```diff
@@ -1,14 +1,14 @@
 name: Format modified connectors
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
 on:
   push:
-    # branches-ignore:
-    #   - master
-    # TODO: remove this once we confirm the workflow is working
-    branches:
-      - "augustin/connectors-ci/python-auto-format"
+    branches-ignore:
+      - master
     paths:
```

Comment:
Will this lead to a race with the test workflow?

For example if I push a commit to my PR wont format and test run at the same time?

If so then I think we have a couple problems
1. Test can get triggered again from the format commit
2. Test could potentially be run before format is applied. Which means were testing old code and might fail the format check

Reply:
Yes it's possible that test are triggered before format. But they will eventually re-run once format is done.
One thing to note is that tests are triggered uppon pull request activities, format workflow are triggered on each push. If developers keep their PR in draft when they iterate the format commit will happen and test will be triggered only when PR is ready for review. 

We could run test and format in the same GHA workflow, but I like the idea of keeping things separate. Wdyt?

Reply:
Hmm I also like the idea of keeping them separate 🤔 

But I also imagine most devs will still go straight to ready for review 😅 

Can we update the GHA that calls test to first call format? Is that an acceptable way to avoid the race and double test issue?